--- "common 1.xsl"	2013-08-19 11:24:23.000000000 +0200
+++ common.xsl	2013-12-31 00:29:07.443696208 +0100
@@ -413,7 +413,7 @@
     <xsl:param name="position">1</xsl:param>
     <xsl:variable name="tokens" select="count($scope/varlistentry)"/>
     <xsl:variable name="token" select="$scope/varlistentry[$position]/term/token"/>
-    <xsl:variable name="size" select="substring-before($token,' KB')"/>
+    <xsl:variable name="size" select="substring-before($token,' Ko')"/>
     <xsl:variable name="rawsize">
       <xsl:choose>
         <xsl:when test="contains($size,',')">
@@ -436,17 +436,17 @@
         <xsl:choose>
           <xsl:when test="$total &lt; '1000'">
             <xsl:value-of select="$total"/>
-            <xsl:text>  KB</xsl:text>
+            <xsl:text>  Ko</xsl:text>
           </xsl:when>
           <xsl:when test="$total &gt; '1000' and $total &lt; '5000'">
             <xsl:value-of select="substring($total,1,1)"/>
             <xsl:text>,</xsl:text>
             <xsl:value-of select="substring($total,2)"/>
-            <xsl:text>  KB</xsl:text>
+            <xsl:text>  Ko</xsl:text>
           </xsl:when>
           <xsl:otherwise>
             <xsl:value-of select="round($total div 1024)"/>
-            <xsl:text>  MB</xsl:text>
+            <xsl:text>  Mo</xsl:text>
           </xsl:otherwise>
         </xsl:choose>
       </xsl:otherwise>
